pkg/capabilities/cli/cmd/testdata/streams.generated.go

```diff
@@ -0,0 +1,128 @@
+// Code generated by pkg/capabilities/cli, DO NOT EDIT.
```

Comment:
This is what will be generated for each capability. Will need work to include capability type, maybe figure out the encoded string => []byte conversion, too.